### DISCUSSION TITLE
Add more severity levels to gauge

### DISCUSSION THREAD

**Discussion by ArdJonker** [home-assistant/frontend#12383] [Dashboard] [answered]
A gauge in some situations needs more severity levels.
![gauge](https://user-images.githubusercontent.com/6270792/164674489-89963351-8005-45e0-9a4d-4a32727c6bde.png)

They could be symmetric (in the case about it is for + or -) but there might also be cases where this is not applicable.

**Comment by zsarnett** [comment, marked as answer]
https://github.com/home-assistant/frontend/pull/11570
